Source/Core/Core/HW/WiimoteEmu/Attachment/Nunchuk.cpp

```diff
@@ -40,6 +41,8 @@ Nunchuk::Nunchuk(ExtensionReg& reg) : Attachment(_trans("Nunchuk"), reg)
 
   // swing
   groups.emplace_back(m_swing = new ControllerEmu::Force(_trans("Swing")));
+  groups.emplace_back(m_swingSlow = new ControllerEmu::Force(_trans("SwingSlow")));
```

Comment:
these.... _are_ supposed to be translated, right...?

Line 35, "C" and "Z" aren't being translated but these are.

Reply:
C and Z are intentionally not translatable because there isn't anything reasonable to translate them to other than C and Z.

Reply:
I guess really, I'm confused over whether this string is for UI display or for loading from config files.

edit: looking at it again, that's what the ConfigInfo structs are for. OK